tf2_py/package.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
-<package>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
```

Comment:
It would be good to change the upstream package.xml to format 2 (and clean it up while we are at it), and then just change the dependencies as necessary here.  Again, if you don't have time to do that, let me know and I can.